file = open("tund8/katsetus.txt", "x")
#kasutades create ehk "x" kÃ¤sklust saab luua
#uue faili, seda aga saab and Ã¼he korra teha
#sest uuesti jooksutades tuleb error, et
#fail eksisteerib juba

